lib/media/drm_engine.js

```diff
@@ -1810,65 +1811,141 @@ shaka.media.DrmEngine = class {
       {contentType: 'video/webm; codecs="vp8"'},
     ];
 
-    const basicConfig = {
-      initDataTypes: ['cenc'],
+    const basicConfigTemplate = {
       videoCapabilities: basicVideoCapabilities,
-    };
-    const offlineConfig = {
-      videoCapabilities: basicVideoCapabilities,
-      persistentState: 'required',
-      sessionTypes: ['persistent-license'],
+      initDataTypes: ['cenc', 'sinf', 'skd'],
```

Comment:
For me it would make sense to test this in a loop to know which initDataTypes are supported, and for you?

Reply:
If we're reporting initDataTypes supported, then yes.  If not, this is fine, because it will succeed if any of these are supported by the key system.

Reply:
Right now there is no use, if we need it in the future, we will add it.